### ISSUE TITLE
Required geometry and centroids

### ISSUE THREAD

**Issue by kiselev-dv** [yohanboniface/geocodejson-spec#11]
There is an issue, when you looking for big admin. boundary or a way, encoding a full geometry into the answer may cause a big overhead, but this is a required field. 

So in many cases, we want to return centroids (or any kind of reference points only). For such purposes, there is centroid field in GeoJSON geomery object. But I think it couldn't be used without coordinates of geometry.

So do someone see the way how not to return centroids instead of full geometries and not to break GeoJSON compatibility?

**Comment by yohanboniface** [comment]
Good point!
I guess that if you return a "Point" geometry, instead of a "Polygon", for an admin boundary, it may be implicit that you are returning the centroid and not the original shape.
But implicit is rarelly a good thing, so I'm not saying this is the ultimate answer to the issue you raised.

**Comment by kiselev-dv** [comment]
Yes, at least, in case you have returned a centroid instead of full geometry as geometry object, it should be clearly marked, that it's not an actual geometry.

**Comment by yohanboniface** [comment]
> Yes, at least, in case you have returned a centroid instead of full geometry as geometry object, it 
> should be clearly marked, that it's not an actual geometry.

Agreed.

@sgillies, any idea on this topic?

**Comment by sgillies** [comment]
Could the result features carry a `geocoding.representative_point = true` property to indicate that the geometry is only a representative point and not the complete geometry of the entity in your database and/or carry a link to the most complete and detailed GeoJSON data for the entity?
